app/src/main/java/xyz/jhughes/laundry/LaundryParser/Machine.java

```diff
@@ -13,6 +13,8 @@
     private String status;
     private String time;
 
+    // TODO: Call constructor when creating machines so stripping leading 0s doesn't
```

Comment:
When you refresh the list or open a new screen basically. getName is still called more often than the constructor though. 

Its a really minor optimization. Probably doesnt matter too much.